acceptance/tests/partitions/partitions_test.go

```diff
@@ -32,10 +32,6 @@ func TestPartitions(t *testing.T) {
 		t.Skipf("skipping this test because -enable-enterprise is not set")
 	}
 
-	if !cfg.UseKind {
-		t.Skipf("skipping this test because Admin Partition tests are only supported in Kind for now")
-	}
-
```

Comment:
Should I add a temporary config value for EKS? To enable this only on Kind and EKS until we get all the clouds running?

Reply:
hmm.. maybe we can explicitly skip AKS and GKE? that way one can peel away those checks as those clouds are configured for these tests? is that possible?

Reply:
should we just consider partitions-eks the feature branch? i saw the aks pr was made against this branch

Reply:
Ya that makes sense, I changed the base to cloud-partitions-tests as the feature branch